lndmanage/lib/liquidityhints.py

```diff
@@ -257,26 +339,33 @@ def penalty(self, node_from: NodeID, node_to: NodeID, edge: Dict, amount_msat: i
         base and relative fees.
         """
         # we assume that our node can always route:
-        if self.source_node in [node_from, ]:
+        if self.source_node in [
```

Comment:
Just curious: Why an inclusion check rather than just `if self.source_node == node_from`?

Reply:
This is a leftover from when I was experimenting with pathfinding (before I checked also `node_to` in the list). Thanks for making me aware.